src/init.cpp

```diff
@@ -1408,7 +1408,14 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
                                  fReindexChainState,
                                  nBlockTreeDBCache,
                                  nCoinDBCache,
-                                 nCoinCacheUsage);
+                                 nCoinCacheUsage,
+                                 []() {
+                                     uiInterface.ThreadSafeMessageBox(
+                                         _("Error reading from database, shutting down."),
+                                         "", CClientUIInterface::MSG_ERROR);
+                                 });
+
+        uiInterface.InitMessage(_("Verifying blocks…").translated);
```

Comment:
In commit "node/chainstate: Decouple from concept of uiInterface" (efe9bd287656925a3bda50a809edeabe9824e37a)

Seems like there is a minor change in behavior with the verifying blocks message being shown even if is_coinsview_empty is true. This seems ok, but would be good to mention whether behavior change is intentional in commit message.